tls/test/test_hello_message.py

```diff
@@ -335,6 +352,75 @@ def test_as_bytes_unsupported_extension(self):
         with pytest.raises(UnsupportedExtensionException):
             record.as_bytes()
 
+    def test_from_bytes_cert_status_request_zero_extension(self):
+        """
+        :py:func:`tls.hello_message.ClientHello.from_bytes` parses a packet
+        with STATUS_REQUEST extension with zero length responder_id_list and
+        request_extensions.
+        """
+        cert_status_request_zero_ext = (
```

Comment:
Why not make this and `client_hello_with_cert_status_request_zero_ext` class variables?